addons/website/data/website_contactus_data.xml

```diff
@@ -148,33 +132,35 @@
         <field name="arch" type="xml">
             <t name="Thanks (Contact us)" t-name="website.contactus_thanks">
                 <t t-call="website.layout">
-                  <div id="wrap">
-                    <div class="oe_structure" id="oe_structure_website_form_contact_us_thanks_1"/>
-                    <div class="container mt-4">
-                        <div class="row">
-                            <div class="col-lg-7 col-xl-6 mr-lg-auto oe_structure">
-                                <section class="pt40 s_text_block pb40 o_colored_level o_cc o_cc1" data-snippet="s_text_block" data-name="Text">
-                                    <div class="container">
-                                        <span class="d-block fa fa-4x fa-thumbs-up mx-auto rounded-circle bg-primary"/><br/>
+                    <div id="wrap" class="oe_structure">
+                        <section class="s_text_block pt40 pb40 o_colored_level " data-snippet="s_text_block">
+                            <div class="s_allow_columns container">
+                                <div class="row">
+                                    <div class="col-lg-7 col-xl-6 mr-lg-auto">
+                                        <span class="d-block fa fa-4x fa-thumbs-up mx-auto rounded-circle bg-primary" />
+                                        <br />
```

Comment:
No need to change but for futur pull request, this is not a mandatory change, as it just creates 2 lines of diff for no real reason
This diff is already a bit hard to analyze (even with hide whitespace changes) so this is the kind of change to avoid :)